ros2launch_security_examples/CMakeLists.txt

```diff
@@ -0,0 +1,95 @@
+cmake_minimum_required(VERSION 3.8)
+project(ros2launch_security_examples)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+
+find_package(ament_nodl REQUIRED)
+find_package(example_interfaces REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rclcpp_components REQUIRED)
+find_package(rclpy REQUIRED)
+find_package(sensor_msgs REQUIRED)
+
+add_library(fake_imu_library SHARED src/fake_imu.cpp)
+ament_target_dependencies(fake_imu_library rclcpp rclcpp_components sensor_msgs example_interfaces)
+rclcpp_components_register_node(fake_imu_library
+  PLUGIN "ros2launch_security_examples::FakeImu"
+  EXECUTABLE fake_imu
+)
+# TODO(wjwwood): go back to organizing these into a nodl folder when this is fixed:
+#   https://github.com/ubuntu-robotics/nodl/issues/41
+# nodl_export_node_description_file(nodl/fake_imu.nodl.xml)
+nodl_export_node_description_file(fake_imu.nodl.xml)
+
+add_library(imu_sink_library SHARED src/imu_sink.cpp)
+ament_target_dependencies(imu_sink_library rclcpp rclcpp_components sensor_msgs example_interfaces)
+rclcpp_components_register_node(imu_sink_library
+  PLUGIN "ros2launch_security_examples::ImuSink"
+  EXECUTABLE imu_sink
+)
+# TODO(wjwwood): go back to organizing these into a nodl folder when this is fixed:
+#   https://github.com/ubuntu-robotics/nodl/issues/41
+# nodl_export_node_description_file(nodl/imu_sink.nodl.xml)
+nodl_export_node_description_file(imu_sink.nodl.xml)
+
+if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+  ament_lint_auto_find_test_dependencies()
+endif()
```

Comment:
This is a duplicate of below.  Looks like a bad merge, maybe?

Reply:
You're right! I think it was left over from `ros2 pkg create`.